build.gradle

```diff
@@ -191,6 +211,24 @@ project('reactor-adapter') {
 
   apply plugin: "biz.aQute.bnd.builder"
 
+	ext {
+		bndOptions = [
+			"Export-Package": [
+				"!*internal*",
+				"reactor.*;version=$osgiVersion;-noimport:=true"
```

Comment:
This might a too broad wildcard. Export-Package means export every package starting with "reactor." on the classpath which means that if this library depends on reactor-core this will also be exported. To prevent that issue you have to use the explicit packages e.g.:
"reactor.adapter.*"

That might also be the reason for the warnings.

An alternative would be exportcontents instruction likes this:
-exportcontents: ${packages}
This ensures that only the packages from the bundle itself are exported but I kind of prefer the Export-Package instruction.

https://bnd.bndtools.org/macros/packages.html